src/Tests.IntegrationTests/ClassifyMessageTests.cs

```diff
@@ -0,0 +1,86 @@
+using Microsoft.Extensions.Configuration;
+using System.Text.Json;
+using Tests.IntegrationTests.Models;
+
+namespace Tests.IntegrationTests
+{
+    public class ClassifyMessageTests
+    {
+        private readonly IConfiguration _configuration;
+
+        public ClassifyMessageTests()
+        {
+            _configuration = new ConfigurationBuilder()
+                .AddJsonFile(@"appsettings.json", false, false)
+                .Build();
+        }
+
+        [Fact(Timeout = 120000)]
+        public async Task GivenValidMessageReceivesValidResponse()
+        {
+            // Arrange
+            using var httpClient = new HttpClient();
+            var bot1Url = _configuration["Bot1Url"];
+            var chatsUri = new Uri($"{bot1Url}/client-api/chats");
+
+            // Act
+            // 1 Create Chat
+            var createdChat = await Request<Chat>(httpClient, Verb.Post, chatsUri).ConfigureAwait(false) as Chat;
+
+            // 2 Create Message
+            var chatMessageUri = new Uri($"{bot1Url}/client-api/chats/{createdChat.Id}/messages");
```

Comment:
I did start to implement this, but I actually think it is cleaner to have build the URI's where they are being used. It is only 2 Uris so not sure there is a lot of value in abstracting it into a helper. If we need to for the next test, then we can refactor as part of the implementation of the next test. I have refactored it slightly to access the base uri directly so that removes 1 line anyway.